tools/operator/launcher/k8s/network.go

```diff
@@ -1,201 +1,547 @@
 package k8s
 
 import (
+	"bytes"
 	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
 
 	"github.com/hyperledger/fabric-test/tools/operator/launcher/nl"
 	"github.com/hyperledger/fabric-test/tools/operator/logger"
-	"github.com/hyperledger/fabric-test/tools/operator/networkclient"
+
 	"github.com/hyperledger/fabric-test/tools/operator/networkspec"
 	"github.com/hyperledger/fabric-test/tools/operator/paths"
-	"github.com/pkg/errors"
+	"github.com/hyperledger/fabric/integration/nwo/fabricconfig"
 )
```

Comment:
A couple accidental blank lines. Be sure to run `go imports -l -w <filename>` after fixing the import statements